.travis.yml

```diff
@@ -14,10 +14,9 @@ before_script:
   - mysql -uroot -e "grant all privileges on otptestssl.* to otptestssl@localhost identified by 'otptestssl' require ssl"
 script: 'make tests'
 otp_release:
-  - 20.0
-  - 19.0
+  - 21.1
+  - 20.3.8.2
+  - 19.3
   - 18.3
   - 17.3
-  - 17.0
   - R16B03-1
-  - R16B
```

Comment:
Why change the oldest version we support? We state in the README that we support versions from R16B. Also, I'd like to know the reason for changing 17.0 -> 17.3, 19.0 -> 19.3 and 20.0 -> 20.3.8.2.

Now, as we don't support all versions from R16B anymore, we should state in the README which versions are supported and which are not, e.g. 21.0 and 20.0-20.2(?)